Homework_1/Question_2.java

```diff
@@ -0,0 +1,26 @@
+import java.io.*;
+
+public class Question_2 {
+    public static void printNumbersDivisibleByThree( int n ) {
+        for( int i = 1; i <= n; i++ ) {
+            if( i % 3 == 0) {
```

Comment:
You had to check whether the sum of digits is divisible by three

Reply:
Ok, will make the changes.

Reply:
Parth, I have made changes. Is it fine, now?